testing_suite/test.sh

```diff
@@ -43,14 +43,16 @@ Running the tests, this might take a while. Please wait...
 
 EOF
 
-newman run https://raw.githubusercontent.com/e2b-dev/agent-protocol/main/testing_suite/contract_tests.json \
--e https://raw.githubusercontent.com/e2b-dev/agent-protocol/main/testing_suite/contract_tests_env.json \
---env-var "env-openapi-json-url=https://raw.githubusercontent.com/e2b-dev/agent-protocol/main/schemas/openapi.json" \
---env-var "env-server=Test server"
+echo "Washington" >> test_output.txt
 
-agent_protocol_contract_testing_results=$?
+newman run https://raw.githubusercontent.com/AI-Engineers-Foundation/agent-protocol/main/testing_suite/agent_protocol_v1.json \
```

Comment:
this URL shows up as empty for me. Is this intended?

Reply:
This URL will exist once it's merged. Otherwise, currently it would be this:

`https://raw.githubusercontent.com/AI-Engineers-Foundation/agent-protocol/9c63b4ce65793459b7ce9b0c1a08df14b7a64cca/testing_suite/agent_protocol_v1.json`

Which is way too long lol

Reply:
Although it may be helpful for future versions to make it belong to a specific commit rather than the main branch at whatever state its in currently.

Only issue is you'd have to know the commit where it worked first before you changed it to that so it's a double commit/merge combo. Maybe it could be automated in a workflow.

Reply:
Good point. Okay sounds good.